react/IntentOpener/styles.styl

```diff
@@ -0,0 +1,30 @@
+@require 'settings/breakpoints'
+
+// TODO remove the second selector when specificity
+// of cozy-ui selectors has decreased
+.intentModal.intentModal
+    // takes 80% of the screen on desktop
+    max-width unset
+    width 80vw
+    height 80vh
```

Comment:
Beware, using `vh`unit is a dangerous game. FYI on Chrome mobile & Safari mobile (if I remember correctly), `100vh` is more than 100% of the actual viewport's height. No problem with `vw` though.

Reply:
OK, I take note of this, if I see any problem, I'll use another technique. Thanks !